Sources/Geotagger/Geotagger.swift

```diff
@@ -30,11 +30,21 @@ public final class Geotagger {
         await withTaskGroup(of: Void.self) { group in
             for item in items {
                 group.addTask { [anchors = self.anchors] in
-                    do {
-                        let tag = try geotagFinder.findGeotag(for: item, using: anchors)
-                        try await item.apply(tag)
-                    } catch let error {
-                        item.skip(with: error)
+                    let result = geotagFinder.findGeotagResult(for: item, using: anchors)
+                    
+                    switch result {
```

Comment:
If we are not handling errors here, let's use try?. Also, we can catch these errors in LoggingGeotaggingItem, print them and then rethrow.